paddle/cuda/src/hl_cuda_cudnn.cc

```diff
@@ -625,18 +633,23 @@ void hl_create_convolution_descriptor(hl_convolution_descriptor* conv,
                                                        padding_width,
                                                        stride_height,
                                                        stride_width,
-                                                       1,
-                                                       1,
+                                                       dilation_h,
+                                                       dilation_w,
                                                        mode,
                                                        data_type));
 #else
+  if (dilation_h > 1 || dilation_w > 1) {
+    LOG(FATAL)
+        << "Current cudnn version does't support for dilation convolution.";
```

Comment:
"Current cuDNN version does't support for dilation convolution.  The dilation convolution requires cuDNN >= v6.0."